src/logic/currencyValues/api/fetchCurrenciesRates.ts

```diff
@@ -1,29 +1,33 @@
 import axios from 'axios'
+import BigNumber from 'bignumber.js'
 
+import { getNetworkInfo } from 'src/config'
 import { EXCHANGE_RATE_URL } from 'src/utils/constants'
-import { AVAILABLE_CURRENCIES } from '../store/model/currencyValues'
 import { fetchTokenCurrenciesBalances } from './fetchTokenCurrenciesBalances'
-import BigNumber from 'bignumber.js'
+import { sameString } from 'src/utils/strings'
+
+const { nativeCoin } = getNetworkInfo()
 
 const fetchCurrenciesRates = async (
-  baseCurrency: AVAILABLE_CURRENCIES,
-  targetCurrencyValue: AVAILABLE_CURRENCIES,
+  baseCurrency: string,
+  targetCurrencyValue: string,
   safeAddress: string,
 ): Promise<number> => {
   let rate = 0
 
-  if (targetCurrencyValue === AVAILABLE_CURRENCIES.ETH) {
+  if (sameString(targetCurrencyValue, nativeCoin.symbol)) {
```

Comment:
Do we need to import nativeCoin here? Shouldn't we still use AVAILABLE_CURRENCIES?
As I can see you added the network token here.
Should we rename it to AVAILABLE_CURRENCIES.NETWORK_TOKEN?

Reply:
super correct!

Reply:
done!